src/main/java/jenkins/plugins/openstack/compute/JCloudsCloud.java

```diff
@@ -389,7 +390,7 @@ public boolean canProvision(final Label label) {
     @Restricted(DoNotUse.class)
     public void doProvision(StaplerRequest req, StaplerResponse rsp, @QueryParameter String name) throws ServletException, IOException,
             Descriptor.FormException {
-        checkPermission(PROVISION);
+        checkPermission(Item.BUILD);
```

Comment:
Please put a comment there pointing to JIRA as this is temporary workaround for a core bug.